input/LocusSpecDecoding.cpp

```diff
@@ -73,9 +73,15 @@ static string extendLocusStructure(
 
     if (numNsInLeftFlank + numNsInRightFlank > maxNsAllowedInFlanks)
     {
-        const string message = "Flanks can contain at most " + to_string(maxNsAllowedInFlanks)
```

Comment:
With the change above, we can keep the original code here.

Reply:
Great, really like that!